wiki/Modding/Kudosu/en.md

```diff
@@ -12,11 +8,15 @@ In [forum modding](/wiki/Modding/Forum_modding), kudosu used to be exchangeable
 
 Nowadays, kudosu is used for the evaluation of a user's capability to [become](/wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator) a [Beatmap Nominator](/wiki/People/Beatmap_Nominators). A user must have 200 kudosu in osu! or 150 kudosu in osu!taiko, osu!catch, or osu!mania in order to apply.
 
-The word *kudosu*, sometimes stylised as *kudosu!*, is a [portmanteau](https://en.wikipedia.org/wiki/Portmanteau) of *kudos* and *osu!*.
+The word *kudosu*, sometimes stylized as *kudosu!*, is a [portmanteau](https://en.wikipedia.org/wiki/Portmanteau) of *kudos* and *osu!*.
+
+### History and Forum Usage
+Originally, kudosu was a way to measure a user's contribution in beatmap discussions, with the goal of improving the quality of the game through modding suggestions. Over time, the kudosu system was refined to more accurately reflect the value of users' contributions. At one point, kudosu was exchanged for [star priority](/wiki/Modding/Star_priority), a benefit that allowed users with more kudosu to get their beatmaps ranked faster, receiving more attention from other members of the community. This system helped accelerate the process of ranking beatmaps so they could be evaluated and eventually included in the game.
+
+### Evaluation of Applications and BN Ranking Systems
+Currently, kudosu plays an important role in evaluating users who want to become a [Beatmap Nominator](/wiki/People/Beatmap_Nominators). To qualify as a BN, a user needs to achieve 200 kudosu in osu! or 150 kudosu in osu!taiko, osu!catch, or osu!mania. Kudosu serves as an indicator of the user's experience and ability to effectively evaluate and modify beatmaps.
```

Comment:
also redundant (the original paragraph with the same info still exists)
"important role" is also quite a stretch if you're actually familiar with the current modding ecosystem